Core/Services/IUserService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using LexiconLMS.Core.Models;
+using LexiconLMS.Core.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LexiconLMS.Core.Services
+{
+    public interface IUserService
```

Comment:
Isn't there something we can extend here that already exists in the framework?

Reply:
It would be possible to extend the UserManager, but as our UserService does not manage users it doesn't really make much sense.

Reply:
Okay. :)